src/Components/ArticleMainCard.jsx

```diff
@@ -36,6 +41,37 @@ function ArticleMainCard() {
     })
   };
 
+////////////////UPLIFTING FROM THE CHILD COMPONENT/////////////////////////////////
+  useEffect(() => {
+    fetchArticleCommentsById(article_id).then((res) => {
+      setComments(res);
+      setIsCommentsLoading(false)
+
+    });
+  }, []);
+
+
+  const handleSubmit=(e)=>{
+    e.preventDefault()
+    if (newComment.length === 0){
+      return e.preventDefault()
+    }
+
+    postComment(article_id,newComment,user.username).then((res)=>{
+      setPostError(false)
+      setNewComment('')
+      setComments((currentComments)=>{ 
+        return [res,...currentComments ]
+    })
```

Comment:
this being in a .then block after the api call means it's not optimistically rendered